packages/zebra_datawedge/example/android/app/build.gradle

```diff
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion flutter.compileSdkVersion
```

Comment:
?

Reply:
Defaults to 35 which requires gradle 8. This is just the example project.